java-manta-it/src/test/java/com/joyent/manta/client/MantaClientAuthenticationIT.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2013-2019, Joyent, Inc. All rights reserved.
```

Comment:
The copyright year should simply be 2019 and not 2013-2019.

Reply:
Noted.